pkg/nsm/apis/netmesh/netmesh.proto

```diff
@@ -26,10 +26,10 @@ import "github.com/ligato/networkservicemesh/pkg/nsm/apis/common/common.proto";
 
 message NetworkServiceChannel {
     common.Metadata metadata = 1;
-    string network_service_name = 2;
-    string nse_provider_name = 3;
+    string networkservicename = 2;
```

Comment:
Curious why changing the name is required?

Reply:
Because this is the error openapi-gen creates if we leave them with the underscores:

```
KMESTERY-M-22GK:networkservicemesh kmestery$ ~/go/bin/openapi-gen --input-dirs ./pkg/apis/networkservicemesh.io/v1 --input-dirs ./pkg/nsm/apis/netmesh --output-package github.com/ligato/networkservicemesh/pkg/apis/networkservicemesh.io/v1 --go-header-file ./conf/boilerplate.txt 
API rule violation: names_match,github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh,NetworkServiceChannel,NetworkServiceName
API rule violation: names_match,github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh,NetworkServiceChannel,NseProviderName
API rule violation: names_match,github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh,NetworkServiceChannel,SocketLocation
F0906 09:28:33.836692   44375 main.go:58] Error: Failed executing generator: some packages had errors:
API rule violations exist
KMESTERY-M-22GK:networkservicemesh kmestery$ 
```

When I change them back to include underscores and regenerate the .go code, you can see that a grep for `NetworkServiceName` in pkg/nsm/apis/netmesh/netmesh.pb.go yields the following:

```
KMESTERY-M-22GK:networkservicemesh kmestery$ grep NetworkServiceName pkg/nsm/apis/netmesh/netmesh.pb.go 
	NetworkServiceName   string              `protobuf:"bytes,2,opt,name=network_service_name,json=networkServiceName,proto3" json:"network_service_name,omitempty"`
func (m *NetworkServiceChannel) GetNetworkServiceName() string {
		return m.NetworkServiceName
```

Reply:
Resolved with latest commit.